backend/infrahub/workers/infrahub_async.py

```
@@ -204,3 +210,35 @@ async def _init_services(self, client: InfrahubClient) -> None:
         )
 
         self.service = service
+
+    async def set_git_global_config(self) -> None:
+        if INFRAHUB_PRODUCTION and not os.getenv("GIT_GLOBAL_CONFIG"):
+            os.makedirs(DEFAULT_GIT_GLOBAL_CONFIG_DIRECTORY, exist_ok=True)  # noqa: PTH103
```

Instead of ignoring the error here we should use Pathlib as suggested by the rule: https://docs.astral.sh/ruff/rules/os-makedirs/